docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,130 @@
+By default, Zowe is reading PKCS12 keystore from `keystore` directory which can be located in zowe.yaml. This directory contains a server certificate, the Zowe generated certificate authority, and a `truststore` which holds intermediate certificates of servers that Zowe communicates with (for example z/OSMF).
 
-## Extended key usage
+The use of a USS PKCS12 keystore is suitable for proof-of-concept projects as special permissions to create and manage the PKCS12 keystore are not required. For production usage of Zowe, it is recommended to work with certificates held in z/OS key rings. Configuring z/OS key rings may require security administrator privileges.  
 
-When a TLS certificate is used for encryption across a socket connection, one of those endpoints is the client and the other is a server.  This usage is restricted with the `Extended Key Usage` (EKU) attribute.  Zowe can work with certificates that have no EKU, however if an EKU is specified it must have both server and client usage. 
+### JCERACFKS certificates in a key ring
 
-An example of where this is a problem with Zowe, is that z/OS certificates may be configured to explicitly only act as server northbound certificates with a `TLS Web Server Authentication (1.3.6.1.5.5.7.3.1)` OID.  However, because Zowe's micro services authenticate to the API Catalog on USS using TLS, the certificate needs to be valid as a southbound client certificate.  To do this, ensure that the certificate contains the `TLS Web Client Authentication (1.3.6.1.5.5.7.3.2)` value in the Extended Key Usage section. 
+Zowe is able to work with certificates held in a **z/OS Key ring**.  
+
+The JCL member `.SZWESAMP(ZWEKRING)` contains security commands to create a SAF key ring. By default, this keyring is named `ZoweKeyring`. You can use the security commands in this JCL member to generate a Zowe certificate authority (CA) and sign the server certificate with this CA. The JCL contains commands for three z/OS security managers: RACF, TopSecret, and ACF/2.
+
+There are two ways to configure and submit `ZWEKRING`:
+
+- Customize and submit the `ZWEKRING` JCL member.
+- Customize the `zowe.setup.certificate` section in `zowe.yaml` and use the `zwe init certificate` command.
+
+Use the `zwe init certificate` command to prepare a customized JCL member using `ZWEKRING` as a template.  
+
+A number of key ring scenarios are supported:
+
+- Creation of a local certificate authority (CA) which is used to sign a locally generated certificate. Both the CA and the certificate are placed in the `ZoweKeyring`.
+- Import of an existing certificate already held in z/OS to the `ZoweKeyring` for use by Zowe.  
+- Creation of a locally generated certificate and signed by an existing certificate authority. The certificate is placed in the key ring.
+
+## Where to get started with Zowe certificates?
+
```

Comment:
To follow our style guide, this header should be rewritten to use gerunds: Getting started with Zowe certificates